gcloud-java-bigquery/src/main/java/com/google/cloud/bigquery/StandardTableDefinition.java

```diff
@@ -252,12 +252,15 @@ ToStringHelper toStringHelper() {
   }
 
   @Override
-  public boolean equals(Object obj) {
-    return obj instanceof StandardTableDefinition && baseEquals((StandardTableDefinition) obj);
+  public final boolean equals(Object obj) {
```

Comment:
If we make it final we are basically saying that users should not mock this one and instead should create an instance with their own pre-defined state in their tests. 

In this particular case, I don't think users can do that. They can create an instance with some of the values but not all. Therefore I would keep it as is (and personally I am fine with keeping its siblings consistent even if some can actually be final).